transformers/reverse/member.ts

```diff
@@ -0,0 +1,33 @@
+import type { Bot } from "../../bot.ts";
+import { DiscordMember, DiscordUser } from "../../types/discord.ts";
+import { Member, User } from "../member.ts";
+
+export function transformUserToDiscordUser(bot: Bot, payload: User): DiscordUser {
+  return {
+    id: bot.utils.bigintToSnowflake(payload.id),
+    username: payload.username,
+    discriminator: payload.discriminator,
+    avatar: payload.avatar ? bot.utils.iconBigintToHash(payload.avatar) : null,
+    locale: payload.locale,
+    email: payload.email ?? undefined,
+    flags: payload.flags,
+    premium_type: payload.premiumType,
+    public_flags: payload.publicFlags,
+    ...payload.toggles.list(),
```

Comment:
Please do this manually, we prevent as much iteration as we can to keep performance at max